pkg/dotc1z/c1file.go

```diff
@@ -39,6 +39,10 @@ func NewC1File(ctx context.Context, dbFilePath string, opts ...C1FOption) (*C1Fi
 		return nil, err
 	}
 	db := goqu.New("sqlite3", rawDB)
+	_, err = db.Exec("PRAGMA synchronous = OFF")
```

Comment:
Should this be a `C1FOption` so we can opt-in to it?

Reply:
I added the option in commit 354a31445213ae855d147ddf55bc704748727ea4